YouTrackAutomation/Functions/Invoke-YTRestMethod.ps1

```diff
@@ -0,0 +1,78 @@
+
+function Invoke-YTRestMethod
+{
+    <#
+    .SYNOPSIS
+    Invokes a REST method in YouTrack.
+
+    .DESCRIPTION
+    The `Invoke-YTRestMethod` function invokes a REST method in YouTrack using the provided session object. Pass in the

```

Comment:
Add information about what the body should be. JSON? Hashtable? Name/value pairs?

Reply:
Bump.